android/build.gradle

```diff
@@ -2,17 +2,18 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "29.0.2"
+        buildToolsVersion = "29.0.3"
         minSdkVersion = 21
-        compileSdkVersion = 29
-        targetSdkVersion = 29
+        compileSdkVersion = 30
+        targetSdkVersion = 30
```

Comment:
Just for reference:

Looks like RN `0.64` still targets `29` (https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.0), but `expo-permissions` (via `react-native-unimodules`) requires `30`. 

Not getting any build errors, so I guess it's fine?